code/datums/gamemodes/game_mode.dm

```diff
@@ -473,6 +474,18 @@ Sensors indicate [numXenosShip || "no"] unknown lifeform signature[numXenosShip
 				continue
 			num_humans++
 
+	for(var/i in GLOB.humans_by_zlevel[ZTRAIT_STATION])
+		var/mob/living/carbon/human/H = i
+		if(!istype(H)) // Small fix?
+			continue
```

Comment:
Just move this junk to distress.dm for the onetime check instead of constantly updating it

Reply:
no